packages/apex-node/src/tests/asyncTests.ts

```diff
@@ -65,14 +68,17 @@ export class AsyncTests {
           sClient.disconnect();
```

Comment:
we can probably move the cancellation handling down, after getting testRunId. Can be a different story.